src/stdune/path.ml

```diff
@@ -1335,3 +1335,16 @@ let chmod ~mode ?(stats = None) ?(op = `Set) path =
         stats.st_perm land lnot mode
   in
   Unix.chmod (to_string path) mode
+
+let realpath t =
```

Comment:
Should we call this function `follow_symlink`, to remove the confusion with the real `realpath`?

Reply:
By the way, what should we do with the symlink chains? Should it be the responsibility of the caller to care about those and possibly iterate?

Reply:
Hmm, I guess we should follow the chains ourselves. This function is added to fix a single use case, so we might as well tailor it to this use case.